src/components/DERP/log.jsx

```diff
@@ -1,37 +1,102 @@
-import React, {useEffect} from "react";
+import React, { useEffect, useState } from "react";
+import styled from "@emotion/styled";
+
+const HighlightedAddress = styled.mark`
+  background-color: #ffa0a0;
+  font-weight: bold;
+`;
+
+const addressesToRegexes = (addresses) => {
+  return addresses.map((address) => {
+    const addressWithoutOx = address.slice(2);
+    return new RegExp(`(0x)?${addressWithoutOx}`, "gi");
```

Comment:
what does this regex do?

Reply:
I'll add jsdoc, thanks for pointing this out